source/core/owt_base/VideoFramePacketizer.cpp

```diff
@@ -14,7 +14,12 @@ static const int TRANSMISSION_MAXBITRATE_MULTIPLIER = 2;
 
 DEFINE_LOGGER(VideoFramePacketizer, "owt.VideoFramePacketizer");
 
-VideoFramePacketizer::VideoFramePacketizer(bool enableRed, bool enableUlpfec, bool enableTransportcc, bool selfRequestKeyframe)
+VideoFramePacketizer::VideoFramePacketizer(
+    bool enableRed,
+    bool enableUlpfec,
+    bool enableTransportcc,
+    bool selfRequestKeyframe,
```

Comment:
What is selfRequestKeyframe for?

Reply:
Previous workaround for some situation, line 291